website/src/blogs/emerging-storage-trends-for-containers.md

```diff
@@ -0,0 +1,53 @@
+
+The storage trends can be summarized as follows:
+
+(1) (Slide#22) **Elastic Storage Infrastructure** — The storage can be horizontally scaled, much like the docker hosts in docker swarm. The technology to implement the Elastic Storage is already in production with many vendors supplying ***Software Defined Scale-out/Distributed Storage*** in the cloud environments.
+
+(2) (Slide #23) **Ease of Accessing the Storage**— The volume of volumes required by the containerized applications will be many fold compared to volumes used in the VM storage. And the volumes will be more portable. Operations team should be shielded from having to mount/unmount volumes or datastores.***The integration into the orchestration layers for mounting the storage and providing access to the volumes, without requiring additional software changes will be paramount.*** The TCMU/iSCSI, NBD/NFS interfaces are two different approaches with each coming with its own nuances w.r.t isolation vs ease.
+
+(3) (Slide #24) **Hyper Converged Storage** — For storage to be hardened, the developer setups need to be and ***avoid managing silos of infrastructure for compute, network and storage.*** The concept of light weight storage software will emerge which will mount distributed external storage or cloud storage on to the local machines (with caching).
+
+(4) (Slide #25) **Hybrid Clouds / Storage** — Put the storage in the right place and move it around depending on the demands of the application and economics. A first in this place would be move the snapshots into S3, when the persistent disks are used for stateful containers. The industry has established that to save money, a mix of clouds are going to be used with the apps moving into different container environments. The storage platform should be able to run along side different clouds or have the ability to inter-operate. ***Seamless Storage Migration within and across clouds is a must.***
+
+(5) (Slide #26) **Containerized Storage** — One of the best innovations is the way the containers are defined and deployed. The storage software also will reap this benefits by containerizing the storage. ***Version of the storage being used will become another parameter to be defined in the intent specs along with capacity and performance (QoS).***Containerization also helps with isolation and ease of upgrades in shared environments.
+
+(5) (Slide #27) **Keep up with the core storage innovations** — Flash is already mainstream, with CIOs questioning if they still really need the SAS Tiers. With NVMe, cache is becoming more accessible both in terms of $$ and the performance boast it can provide to the Storage Software. ***The low latency demands from the containerized applications will be guaranteed via the using of NVMe Flash for read/write caching of data that is possible stored on disks or remote cloud storage.***
```

Comment:
```suggestion
(5) (Slide #27) **Keep up with the core storage innovations** — Flash is already mainstream, with CIOs questioning if they still really need the SAS Tiers. With NVMe, cache is becoming more accessible both in terms of $$ and the performance boost it can provide to the Storage Software. ***The low latency demands from the containerized applications will be guaranteed via the use of NVMe Flash for read/write caching of data that is possibly stored on disks or remote cloud storage.***
```